test/test.js

```diff
@@ -1777,6 +1777,23 @@ describe('Page', function() {
     });
   });
 
+  describe('ElementHandle.boxModel', function() {
+    it('should work', async({page, server}) => {
+      await page.setViewport({width: 500, height: 500});
+      await page.goto(server.PREFIX + '/grid.html');
+      const elementHandle = await page.$('.box:nth-of-type(13)');
+      const box = await elementHandle.boxModel();
+
+      expect(box).not.toBe(null);
+      expect(box.content).not.toBe(null);
```

Comment:
let's check exact values.

Can you also check a rotated element? This is to make sure our protocol method works as expected.